R/module_teal.R

```diff
@@ -25,6 +25,11 @@
 #'
 #' It is written as a `shiny` module so it can be added into other apps as well.
 #'
+#' @section Reproducibility:
+#' Reproducibility is supported by multiple features. `teal` includes a `utils::sessioInfo()` output to allow to compare
+#' packages used in the session. It also allows to create `renv` `.lockfile` to support project setup reproducibility.
+#' For more information about `.lockfile` creation visit [`create_renv_lockfile()`].
+#'
```

Comment:
Should we move this section to `module_teal_with_splash` since it is used for internal functions?` module_teal_with_splash `is exported.

Reply:
hmmm, or maybe inside `teal::init`?

Reply:
This might be confusing because it's not related to any parameter passed to init. Maybe we should keep it in `teal_module_with_splash`, but let's either create a vignette for it or add it to the blueprint and explain its details. We should explain to the user how they can keep the renv.lock file and how this feature works. WDYT?

Reply:
yeah, I think we could move to `teal_module_with_splash`. I don't think an extra vignette is needed though as this is a side-feature

Reply:
moved in here 72c98b557531949ac335810d67483e176dd58e5d